source/whats-new.txt

```diff
@@ -12,12 +12,34 @@ What's New
 
 Learn what's new in:
 
+* :ref:`Version 4.4 <version-4.4>`
 * :ref:`Version 4.3 <version-4.3>`
 * :ref:`Version 4.2 <version-4.2>`
 * :ref:`Version 4.1 <version-4.1>`
 * :ref:`Version 4.0 <version-4.0>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-4.4:
+
+What's New in 4.4
+-----------------
+
+New features of the 4.4 Node.js driver release include:
+
+- KMIP support when using CSFLE.
+- TLS support when using CSFLE.
+- Hostname canonicalization now accepts "none", "forward", and "forwardAndReverse" as ``authMechanismProperties`` when using GSSAPI.
+- In the 4.0.0 release of the driver, the deprecated ``collection.count()`` method was inadvertently changed to behave like ``collection.countDocuments()``.
+  In this release, the ``collection.count()`` method is updated to match legacy behavior:
+
+  - If a query is passed in, ``collection.count()`` will behave the same as ``collection.countDocuments()`` and perform a collection scan.
+  - If no query is passed in, ``collection.count()`` will behave the same as ``collection.estimatedDocumentCount()`` and rely on collection metadata..
+
+  .. important:: Deprecation Notice
+
+     The ``cursor.count()`` method is deprecated and will be removed in the next major version, along with ``collection.count()``.
```

Comment:
Comment: Unsure which "next major version" is being referred to. Is this the Node.js driver, MongoDB Server, or both? My guess is driver (and possibly server), but this may be unclear to readers.